pro/src/components/IndividualOffer/SummaryScreen/SummaryScreen.spec.tsx

```diff
@@ -0,0 +1,321 @@
+      hours: 10,
+      minutes: 0,
+    })
+
+    it("should validate publication date and time when it's a scheduled publication", async () => {
+      renderSummaryScreen({ contextValue })
+
+      await userEvent.click(
+        screen.getByLabelText(LABELS.publicationModeLaterRadio)
+      )
+      await userEvent.click(
+        screen.getByRole('button', { name: LABELS.submitScheduledOfferButton })
+      )
+
+      expect(
+        await screen.findByText(ERROR_MESSAGES.publicationDateIsRequired)
+      ).toBeVisible()
+      expect(
+        await screen.findByText(ERROR_MESSAGES.publicationTimeIsRequired)
+      ).toBeVisible()
+
+      const publicationDateInput = screen.getByLabelText(
+        LABELS.publicationDateInput
+      )
+      const publicationTimeSelect = screen.getByLabelText(
+        LABELS.publicationTimeSelect
+      )
+
+      await userEvent.type(
+        publicationDateInput,
+        format(inOneMonth, 'yyyy-MM-dd')
+      )
+
+      expect(
+        screen.queryByText(ERROR_MESSAGES.publicationDateIsRequired)
+      ).not.toBeInTheDocument()
+      expect(
+        screen.queryByText(ERROR_MESSAGES.publicationDateMustBeInFuture)
+      ).toBeVisible()
+      expect(
+        screen.queryByText(ERROR_MESSAGES.publicationTimeIsRequired)
+      ).toBeVisible()
+
+      await userEvent.selectOptions(
+        publicationTimeSelect,
+        format(inOneMonth, 'HH:mm')
+      )
+
+      expect(
+        screen.queryByTestId('error-publicationDate')
+      ).not.toBeInTheDocument()
+
+      await userEvent.click(
+        screen.getByRole('button', { name: LABELS.submitScheduledOfferButton })
+      )
+
+      expect(api.patchPublishOffer).toHaveBeenCalledWith({
+        id: contextValue.offer!.id,
+        publicationDatetime: expect.any(String),
+        bookingAllowedDatetime: undefined,
+      })
+    })
+
+    it("should require publication date to be in the future when it's a scheduled publication", async () => {
+      const yesterday = set(sub(new Date(), { days: 1 }), {
+        hours: 10,
+        minutes: 0,
+      })
+
+      renderSummaryScreen({ contextValue })
+
+      await userEvent.click(
+        screen.getByLabelText(LABELS.publicationModeLaterRadio)
+      )
+
+      const publicationDateInput = screen.getByLabelText(
+        LABELS.publicationDateInput
+      )
+      const publicationTimeSelect = screen.getByLabelText(
+        LABELS.publicationTimeSelect
+      )
+
+      await userEvent.type(
+        publicationDateInput,
+        format(yesterday, 'yyyy-MM-dd')
+      )
+      await userEvent.selectOptions(
+        publicationTimeSelect,
+        format(yesterday, 'HH:mm')
+      )
+
+      expect(
+        await screen.findByText(ERROR_MESSAGES.publicationDateMustBeInFuture)
+      ).toBeVisible()
+
+      await act(() =>
+        fireEvent.input(publicationDateInput, {
+          target: {
+            value: format(inOneMonth, 'yyyy-MM-dd'),
+          },
+        })
```

Comment:
Pas moyen d'utiliser `userEvent` à la place ?
```suggestion
      await userEvent.clear(publicationDateInput)
      await userEvent.type(publicationDateInput, format(inOneMonth, 'yyyy-MM-dd'))
```